geoh5py/workspace/workspace.py

```diff
@@ -827,6 +827,13 @@ def objects(self) -> list[objects.ObjectBase]:
         return self._all_objects()
 
     def open(self, mode: str | None = None):
+        if isinstance(self._geoh5, h5py.File) and self._geoh5:
+            warnings.warn(
+                f"Workspace already opened in mode {self._geoh5.mode}. "
+                "Consider closing the file first with 'close()'."
```

Comment:
**suggestion:** not sure this second message line is necessary.
If the file is already open, there is no need to close it.
Unless the file is open as read-only and it tries to open read/write. But in that case, I assume there would be an error already.

Reply:
That's the case I had in mind yes. I am fine with removing the hint.